.eslintrc.js

```diff
@@ -2063,6 +2063,7 @@ module.exports = {
      */
     {
       files: [
+        'src/platform/packages/*/kbn-{package-*,repo-*,dep-*}/**/*',
```

Comment:
What are these patterns covering? do we have packages that are literally named `kbn-package-...`?

Reply:
Haven't checked TBH, the patterns were already there, so I just added them for the new path too (as I'm moving ~half of the packages under `/packages`). Will take a look

Reply:
Removed them on https://github.com/elastic/kibana/pull/208704/commits/63fa1c06ca01dc5842af9ab625ff8de67e0d5f89